shared/constants/signup.js

```diff
@@ -1,23 +1,30 @@
 // @flow
-import type {State} from './types/signup'
-import {isMobile} from '../constants/platform'
+import * as I from 'immutable'
+import * as Types from './types/signup'
+import {isAndroid, isIOS, isDarwin, isWindows, isLinux, isMobile} from '../constants/platform'
 
-const initialState: State = {
-  deviceName: isMobile ? 'Mobile Device' : 'Home Computer',
-  deviceNameError: null,
-  email: null,
-  emailError: null,
-  inviteCode: null,
-  inviteCodeError: null,
-  nameError: null,
+const devicename =
```

Comment:
lets default to platform specific names.
@malgorithms have any opinions about the verbiage?

Reply:
nah